common/changes/lydutk-fixedPersonaSize28InitialsHeight_2017-05-04-15-28.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Fixed Persona size28 height of initials",
```

Comment:
The standard way of phrasing this is "<Component>: <Description of fix>". Also, it's included directly in the release notes, so I'd have it describe the change in behavior, rather than the change in code. A possible comment: 
````
Persona: Fixed clipping issue when using size28.
````